apps/codelab/src/app/codelabs/extra/feedback-page/feedback-page.component.html

```diff
@@ -1,69 +1,61 @@
-<slide-deck slideShortcuts slidesRouting kirjs-tracking>
+<slide-deck slideShortcuts slidesRouting kirjs-tracking class="content">
```

Comment:
Is this file even needed anymore, now when you moved everything to the admin panel?